### DISCUSSION TITLE
Enhancement: Spookeez overhaul

### DISCUSSION THREAD

**Discussion by NickHam13** [FunkinCrew/Funkin#1586] [Ideas]
Not really sure whether this counts as a bug report or an enhancement but the game is technically working as intended, so I'll go with enhancement.

The first song of Week 2, Spookeez, has two glaring issues:
1. The voice track is slightly ahead of the music track. For what it's worth, this isn't present in the HTML5 versions of the game but in all other footage I've seen of the song being played, it's desynced.
2. The charts are very inaccurate. A lot of mistimed notes, missing notes, extra notes and some other funky stuff that makes it hard to FC. It was only the second week but then again, South doesn't have these issues so it should still be revised.

I submitted [a mod](https://gamebanana.com/gamefiles/14667) to GameBanana that fixes both of these issues but I would still like some sort of official fix for this. I thought about making a pull request with these changes but it feels a bit weird or improper to PR a mod. Let me know if I should follow up on that.

**Comment by regulad** [comment]
> I thought about making a pull request with these changes but it feels a bit weird or improper to PR a mod.

I mean, if it actually fixes a valid issue there is no harm in making a PR.

**Comment by NickHam13** [comment]
> > I thought about making a pull request with these changes but it feels a bit weird or improper to PR a mod.
> 
> I mean, if it actually fixes a valid issue there is no harm in making a PR.

Fair enough, I'll look into it.

**Comment by xChillax** [comment]
I fixed it in the past, submitted it and I was told he was going to fix it by himself eventually, since he wasn't happy with the original charting he made for it and wanted to remake it on his own.
